programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,20 @@
+function solution(numbers) {
+  let answer = [];
+  let uniqueAnswer = [];
+
+  for (let i = 0; i < numbers.length - 1; i++) {
+    for (let j = i + 1; j < numbers.length; j++) {
+      answer.push(numbers[i] + numbers[j]);
+    }
+  }
+
+  answer = answer.sort((a, b) => {
+    return a - b;
+  });
+
+  uniqueAnswer = answer.filter((element, index) => {
+    return answer.indexOf(element) === index;
+  });
+
+  return uniqueAnswer;
```

Comment:
```suggestion
   return answer.sort((a, b) => a - b).filter((element, index) => answer.indexOf(element) === index);
```
sort와 filter를 사용하셨는데 잘 사용하셨어요!

문제를 푸실 때 answer에 answer를 정렬한 값을 재 할당 하셨고 그 배열에 다시 filter를 적용하려고 하신 것 같아요.
메서드를 여러번 사용하고 싶다면 지금처럼 여러 줄에 걸쳐서 할 수도 있지만,
제가 한 것처럼 메서드를 이어서 사용할 수도 있어요. 이런 방식을 메서드 체이닝이라고 합니다.

이렇게 하면 uniqueAnswer라는 배열은 선언 안해도 되겠죠?

[메서드 체이닝](https://velog.io/@seungsang00/JavaScript-Method-Chaining)